src/zoid/qr-code/component.jsx

```diff
@@ -185,7 +195,7 @@ export function QRCodeContainer({
                 box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.4);
                 border-radius: 16px;                        
                 width: 720px;
-                height: 544px;  
+                height: 612px;  
```

Comment:
@andregardi Why are we changing the dimensions of the QRCodeContainer?

Reply:
We had a redesign last week on the size of the modal:
https://www.figma.com/file/g9Pg0vQJK0EVeiBLiuMoC2/Universal-Venmo-Pay?node-id=7269%3A33206

Now that I am revisiting that, this new modal height is only `612px` if the survey is displayed. I am sorry for that, I got confused because the design kept changing. I am not sure yet how could we implement this dynamic height controlled by the child iframe. If you have any suggestion on that, let me know!

Reply:
Style it so that there is a min-height and max-height and let the container expand for survey.

Reply:
Thank you for the suggestion @mnicpt ,
But that is not so easy in that case.
Between the `containerTemplate` and the `QRCard` there is an iframe. I believe that there is no CSS property that allow the iframe to resize according to its content. That means, expand the survey or any other container on SPB would not affect the size of the `containerTemplate` here.